server/curatorship/src/main/kotlin/br/usp/inovacao/hubusp/curatorship/sheets/Campus.kt

```diff
@@ -0,0 +1,143 @@
+package br.usp.inovacao.hubusp.curatorship.sheets
+
+enum class Campus(
+    private val printableName: String? = null
+) {
+
+    Bauru {
+        override fun unities(): Set<String> = setOf(
+            "Faculdade de Odontologia de Bauru - FOB",
+            "Hospital de Reabilitação de Anomalias Craniofaciais - HRAC"
+        )
+    },
+
+    Butanta("Butantã") {
+        override fun unities(): Set<String> = setOf(
+            "Escola de Comunicações e Artes - ECA",
+            "Escola de Educação Física e Esporte - EEFE",
+            "Escola Politécnica - EP",
+            "Faculdade de Arquitetura e Urbanismo - FAU",
+            "Faculdade de Ciências Farmacêuticas - FCF",
+            "Faculdade de Economia, Administração e Contabilidade - FEA",
+            "Faculdade de Educação - FE",
+            "Faculdade de Filosofia, Letras e Ciências Humanas - FFLCH",
+            "Faculdade de Medicina Veterinária e Zootecnia - FMVZ",
+            "Faculdade de Odontologia - FO",
+            "Hospital Universitário - HU",
+            "Instituto de Astronomia, Geofísica e Ciências Atmosféricas - IAG",
+            "Instituto de Biociências - IB",
+            "Instituto de Ciências Biomédicas - ICB",
+            "Instituto de Energia e Ambiente - IEE",
+            "Instituto de Estudos Avançados - IEA",
+            "Instituto de Estudos Brasileiros - IEB",
+            "Instituto de Física - IF",
+            "Instituto de Geociências - IGc",
+            "Instituto de Matemática e Estatística - IME",
+            "Instituto de Psicologia - IP",
+            "Instituto de Química - IQ",
+            "Instituto de Relações Internacionais - IRI",
+            "Instituto Oceanográfico - IO",
+            "Instituto de Pesquisas Energéticas e Nucleares - IPEN"
+        )
+    },
+
+    SaoSebastiao("Butantã / São Sebastião") {
```

Comment:
Butantã / São Sebastião is the old version, needs to overwrite by São Sebastião (Butantã)